sdk/spring/spring-cloud-azure-appconfiguration-config/pom.xml

```diff
@@ -83,12 +83,6 @@
       <version>3.3.3</version> <!-- {x-version-update;springboot3_org.springframework.boot:spring-boot-starter-test;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Why do we remove this one?

Reply:
Pipeline failed due to the duplicate dependency, cc: @mrm9084

Reply:
Duplicated with line 26 - line 31